lib/core/src/client/preview/loadCsf.ts

```diff
@@ -113,24 +113,29 @@ const loadStories = (
       kind.addDecorator(decorator);
     });
 
-    Object.keys(exports).forEach((key) => {
-      if (isExportStory(key, meta)) {
-        const storyFn = exports[key];
-        const { name, parameters, decorators, args, argTypes } = storyFn.story || {};
-        const decoratorParams = decorators ? { decorators } : null;
-        const exportName = storyNameFromExport(key);
-        const idParams = { __id: toId(componentId || kindName, exportName) };
-
-        const storyParams = {
-          ...parameters,
-          ...decoratorParams,
-          ...idParams,
-          args,
-          argTypes,
-        };
-        kind.add(name || exportName, storyFn, storyParams);
-      }
-    });
+    const storyExports = Object.keys(exports);
+    if (storyExports.length === 0) {
+      logger.warn(`Found a storyfile for "${kindName}" but no exported stories.`);
```

Comment:
Let me know if there is a better message for this! Maybe a link to the [docs](https://storybook.js.org/docs/formats/component-story-format/)? 🤔

Reply:
Yeah, docs link could be good.

Also, personal preference would be to just return from the function here & not have an `else` statement, which makes the file nesting deeper than it needs to be.

```
if(exception case) {
  some simple logic
  return;
}
some complex logic;
```

Reply:
Great suggestion. Done!